fe1-web/src/features/rollCall/network/RollCallHandler.ts

```diff
@@ -167,14 +167,42 @@ export function handleRollCallCloseMessage(msg: ProcessableMessage): boolean {
 }
 
 /**
- * TODO: Handles a reopen roll call message.
+ * Handles a RollCallReopen message by reopening the corresponding roll call.
  *
- * @param msg
+ * @param msg - The extended message for reopening a roll call
  */
 export function handleRollCallReopenMessage(msg: ProcessableMessage) {
-  console.warn(
-    'A RollCall reopen message was received but its processing logic is not yet implemented:',
-    msg,
-  );
-  return false;
+  if (
+    msg.messageData.object !== ObjectType.ROLL_CALL ||
+    msg.messageData.action !== ActionType.REOPEN
+  ) {
+    console.warn('handleRollCallReopenMessage was called to process an unsupported message', msg);
+    return false;
+  }
+
+  const makeErr = (err: string) => `roll_call/reopen was not processed: ${err}`;
+
+  const storeState = getStore().getState();
+  const lao = getCurrentLao(storeState);
+  if (!lao) {
+    console.warn(makeErr('no LAO is currently active'));
+    return false;
+  }
+
+  const rcMsgData = msg.messageData as ReopenRollCall;
+  const oldRC = getEventFromId(storeState, rcMsgData.opens) as RollCall;
+  if (!oldRC) {
```

Comment:
Would it make sense to (defensively) check if `oldRc.status` is `CLOSED`, since we're re-opening ?

Reply:
done !